syntax = "proto3";

message KeyValuePair {
  string key = 1;
  string value = 2;
}

message ListPairs {
  repeated KeyValuePair pairs = 1;
}

message StatusResponse {
  bool ok = 1;
}

message Empty {
    
}

service DictService {
  rpc AddPair(KeyValuePair) returns (StatusResponse);
  rpc GetPairs(Empty) returns (ListPairs);
}
